changes/6181-mark-todd.md

```diff
@@ -0,0 +1 @@
+Updated docs to show frozen can be used instead of allow_mutation
```

Comment:
We don't need this file for changes like this
Please remove it

Reply:
File removed